### PR TITLE
Java: add MOVE command

### PR DESCRIPTION
**Pull request by aaron-congo**
https://redis.io/docs/latest/commands/move/

NOTE: MOVE is not supported with cluster clients since, unlike standalone clients, cluster clients do not support databases

### CODE REVIEW

**Comment by GumpacG** [comment]
Could you add a test where there are two databases and a key exists in db2 and db1. Move the key to db2 but returns false since it exists in db2 already?

File: java/integTest/src/test/java/glide/standalone/CommandTests.java [hunk 1]
```diff
@@ -154,6 +154,25 @@ public void select_test_gives_error() {
...
+    @SneakyThrows
+    public void move() {
+        String key = UUID.randomUUID().toString();
+        String value = UUID.randomUUID().toString();
+        String nonExistingKey = UUID.randomUUID().toString();
+        assertEquals(false, regularClient.move(nonExistingKey, 1L).get());
+
+        assertEquals(OK, regularClient.set(key, value).get());
+        assertEquals(true, regularClient.move(key, 1L).get());
+        assertEquals(OK, regularClient.select(1).get());
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/standalone/CommandTests.java:167]: Try getting the `key` between `move` and `select` - should return `null`

File: java/integTest/src/test/java/glide/standalone/CommandTests.java [hunk 2]
```diff
@@ -154,6 +154,25 @@ public void select_test_gives_error() {
...
+        assertEquals(value, regularClient.get(key).get());
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/standalone/CommandTests.java:168]: Add test when moving over existing key

File: java/integTest/src/test/java/glide/standalone/TransactionTests.java [hunk 3]
```diff
@@ -126,10 +126,10 @@ public void test_standalone_transaction() {
         String key = UUID.randomUUID().toString();
         String value = UUID.randomUUID().toString();
 
-        Transaction transaction =
-                new Transaction().select(1).set(key, value).get(key).select(0).get(key);
+        Transaction transaction = new Transaction();
+        transaction.set(key, value).get(key).move(key, 1L).get(key).select(1).get(key);
 
-        Object[] expectedResult = new Object[] {OK, OK, value, OK, null};
+        Object[] expectedResult = new Object[] {OK, value, true, null, OK, value};
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/standalone/TransactionTests.java:132]: Can you please place 1 respose per line and add comments to which command they belong to?

**Review by GumpacG — APPROVED** [review]

**Pull request merged**